decision records/003-Depenedency-Manager.md

```diff
@@ -0,0 +1,25 @@
+# Dependency Manager 
+•	Status: Proposed
+•	Issue: #638
+
+## Context and Problem Statement 
+As the number of packages increases in large scale projects, with multiple dependencies, updating package versions and dependencies manually is time consuming and takes great care to update dependencies one by one. 
+
+Other solutions encountered were local bash scripts as documented in both the issue https://github.com/Accenture/sfpowerscripts/issues/638 raised and in the discussions has here: https://github.com/Accenture/sfpowerscripts/discussions/468 
+
+## Solution
+The dependency manager should utilise the SemVer incrementation style https://semver.org/ and be of a similar method to the ‘lerna version’ command. It should prompt the user for the version: (major, minor, patch, custom). 
```

Comment:
Lerna has a no prompt support as well, where we say Major it updates major for all packages

Reply:
Updated